### PR TITLE
include semantic release

### PR DESCRIPTION
**Pull request by bluebill1049**
semantic release

### CODE REVIEW

File: .releaserc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (jorisre) [inline .releaserc:1]: Not needed, that's the default configuration: https://semantic-release.gitbook.io/semantic-release/usage/configuration#branches

Comment (bluebill1049) [reply .releaserc]: ty 👍

**Review by jorisre — APPROVED** [review]

**Comment by bluebill1049** [comment]
ty @jorisre

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      - name: Install deps and build (with cache)
+        uses: bahmutov/npm-install@v1
+
+      - name: Build
+        run: yarn build
+
+  publish-module:
+    name: 'Publish Module to NPM on Node ${{ matrix.node }} and ${{ matrix.os }}'
+    needs: build
+    if: github.repository == 'little-state-machine'  && (github.ref == 'refs/heads/master')
```
Comment (bluebill1049) [inline .github/workflows/main.yml:36]: @jorisre looks like the check failed. any idea?

**Pull request merged**